frontend/packages/client/src/components/Community/StrategyEditorModal/index.js

```diff
@@ -84,6 +60,26 @@ export default function StrategyEditorModal({
 
   // user selected strategy move to second step to enter information
   const setStrategy = (strategyName) => {
+    //
+    // STRATEGY CONFIGURATION
+    //
+    // If strategy selected is 'one-address-one-vote'
+    // then no more information is required(FE uses strategy configuration)
+    // modal should be closed and strategy should be ready to be added
+    if (strategyName === 'one-address-one-vote') {
+      const strategyConfig =
+        networkConfig.strategiesConfig['one-address-one-vote'];
+      onDone({
+        name: strategyName,
+        contract: {
+          name: strategyConfig.name,
+          addr: strategyConfig.addr,
+          publicPath: strategyConfig.publicPath,
+          threshold: '0',
+        },
+      });
+      return;
+    }
```

Comment:
this won't allow users to enter custom information on one-address-one-vote. Is this correct?

Reply:
This will be removed - accidentally got put back in when testing with branch with backend changes

Reply:
Missed this - it will be removed since we will not support one-address-one-vote